packages/wallet/api/test/test-wallet-marshal.js

```diff
@@ -163,4 +163,19 @@ test('makeExportContext.serialize handles unregistered identites', t => {
     slots: ['payment:1'],
   });
   t.deepEqual(context.unserialize(cap2), myPayment);
+
+  {
+    const val = Far('someOfferResult', {});
+    context.saveOfferResultActions(val);
+    const cap = context.serialize(val);
+    t.deepEqual(cap, {
+      body: JSON.stringify({
+        '@qclass': 'slot',
+        iface: 'Alleged: someOfferResult',
+        index: 0,
+      }),
+      slots: ['offerResult:1'],
+    });
+    t.deepEqual(context.unserialize(cap), val);
```

Comment:
it's important that `unserialize` preserves identity, so this test should be `t.is(...)`